src/views/AccountDetail.vue

```diff
@@ -20,7 +20,7 @@
         </template>
       </Card>
 
-      <Card v-if="1||showMultiSigDeatail" class="card-f card-adaptive" :loading="loading">
```

Comment:
i think `showMultiSigDeatail` is valild because it existing in `computed` , only `1||` have to remove.